README.md

```diff
@@ -1 +1,43 @@
 # salmon-sync
+
+This is a Github Action that syncs a folder to a Google Cloud bucket using `rclone`.
+This action is only meant to work for Deephaven's documentation. It could be used in a more general purpose way to sync a folder into any Google cloud bucket (with the proper credentials), but that is subject to change and may break in any version.
+
+## Parameters
+```
+inputs:
+  source:
+    required: true
+    type: string
+    description: 'The source directory to sync.'
+  destination:
+    required: true
+    type: string
+    description: 'The destination directory to sync. Relative to the bucket.'
+  project_number:
+    required: true
+    type: string
+    description: 'The Google Cloud project number.'
+  bucket:
+    required: true
+    type: string
+    description: 'The Google Cloud bucket to sync to. It is recommended to use the GitHub repo path (such as deephaven/salmon-sync) as the minimum base to prevent collisions.'
```

Comment:
I think the 2nd sentence of this should be on `destination` not `bucket`. Same for the yml file

Reply:
Fixed